MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication.swift

```diff
@@ -151,24 +151,18 @@ public final class MSALNativeAuthPublicClientApplication: MSALPublicClientApplic
 
     /// Sign up a user with a given username and password.
```

Comment:
We should mention that we use a parameter here.
```suggestion
    /// Sign up a user for a provided parameters.
```